### ISSUE TITLE
ClickHouse: LIMIT n BY with multiple columns is unparsable

### ISSUE THREAD

**Issue by friedrich123456789** [sqlfluff/sqlfluff#8030]
### What happened

`LIMIT n BY a` parses, but a comma-separated column list `LIMIT n BY a, b`
leaves the trailing column(s) unparsable.

This is a follow-up to #5494 (single-column `LIMIT BY`, fixed) — the multi-column
form still fails.

### Expected behaviour

`LIMIT n BY <expr> [, <expr>]...` accepts a list of expressions.

### Observed behaviour

```
SELECT * FROM t LIMIT 1 BY a, b  ->  L1:P29 PRS: Found unparsable section: ', b'
SELECT * FROM t LIMIT 1 BY a     ->  parses fine
```

### How to reproduce

```sql
SELECT * FROM t LIMIT 1 BY a, b
```

```bash
sqlfluff parse --dialect clickhouse -
```

Valid, verified on ClickHouse 26.4.2.
Docs: https://clickhouse.com/docs/sql-reference/statements/select/limit-by

### Dialect

clickhouse

### Version

sqlfluff 4.2.2 (latest release), Python 3.12; also applies to `main`.

### Configuration

None — default rules, `--dialect clickhouse`.
